src/connector/codec/src/decoder/avro/mod.rs

```diff
@@ -318,6 +318,29 @@ impl<'a> AvroParseOptions<'a> {
             (DataType::Varchar, Value::Uuid(uuid)) => {
                 uuid.as_hyphenated().to_string().into_boxed_str().into()
             }
+            (DataType::Map(map_type), Value::Map(map)) => {
```

Comment:
LGTM. BTW do you know why we use apache-avro 0.16.0 instead of 0.17.0 here in this crate?

Reply:
- We have a few bugfix patches in our fork https://github.com/risingwavelabs/avro/issues/6
- And 0.17.0 is just released last week https://github.com/risingwavelabs/avro/issues/5